packages/clerk-js/src/ui/elements/Menu.tsx

```diff
@@ -17,24 +19,28 @@ type MenuState = {
   elementId?: MenuId;
 };
 
-const [MenuStateCtx, useMenuState] = createContextAndHook<MenuState>('MenuState');
+export const [MenuStateCtx, useMenuState] = createContextAndHook<MenuState>('MenuState');
 
-type MenuProps = PropsWithChildren<Record<never, never>> & { elementId?: MenuId };
+type MenuProps = PropsWithChildren<Record<never, never>> & {
+  elementId?: any;
+  popoverPlacement?: Placement;
+};
 
 export const Menu = withFloatingTree((props: MenuProps) => {
-  const { elementId } = props;
+  const { popoverPlacement = 'bottom-end', elementId, ...rest } = props;
```

Comment:
Shouldn't we use `right-start` as a default like before?

Reply:
With `right-start` some place we where using the Menu like the actions on the `<UserProfile/> Email addresses
section `, the popover was floating outside the component, so by using `bottom-end` looked a bit nicer.

With `right-start`:
![CleanShot 2024-01-05 at 21 37 47](https://github.com/clerk/javascript/assets/6823226/f7ebdbe4-baa1-4fca-8819-676aa3d08ff8)

With `bottom-end`:
![CleanShot 2024-01-05 at 21 39 02](https://github.com/clerk/javascript/assets/6823226/f96d7cce-4d50-4920-8865-793eccc6f09d)